legal-api/src/legal_api/models/business.py

```diff
@@ -15,7 +15,7 @@
 
 The Business class and Schema are held in this module
 """
-from datetime import datetime
+from datetime import datetime, timezone
```

Comment:
Please move this to using https://github.com/bcgov/lear/blob/master/legal-api/src/legal_api/utils/datetime.py#L21
since you are in here.

Reply:
I tried doing this but it doesn't seem to recognize the class method so leaving this seems to be the most prudent way to keep it working

Reply:
it's already used in the codebase, feel free to search for examples
```
from legal_api.utils.datetime import datetime, timezone
```

Reply:
I have no idea what I was doing wrong but yea it works now, change made